src/sphinx_book_theme/nodes.py

```diff
@@ -0,0 +1,48 @@
+from docutils import nodes
+from sphinx.application import Sphinx
+from typing import Any, cast
+from sphinx.writers.latex import LaTeXTranslator
+
+
+class SideNoteNode(nodes.Element):
+    """A node that will not be rendered."""
+
+    def __init__(self, rawsource="", *children, **attributes):
+        super().__init__("", **attributes)
+
+    @classmethod
+    def add_node(cls, app: Sphinx) -> None:
+        add_node = cast(Any, app.add_node)  # has the wrong typing for sphinx<4
+        add_node(
+            cls,
+            override=True,
+            html=(visit_SideNoteNode, depart_SideNoteNode),
+            latex=(visit_SideNoteNode, depart_SideNoteNode),
+        )
+
+
+def visit_SideNoteNode(self, node):
+    if isinstance(self, LaTeXTranslator):
```

Comment:
The post transform should be set to only run on the html format, then you can get rid of this

Reply:
sounds good. The way to handle it in latex should probably be in `sphinx-jupyterbook-latex`? At present, the pdf just ignores margin texts. 
Unless we decide to move this functionality out of `sphinx-book-theme`

Reply:
Yep exactly 👍, whilst it is only in sphinx-book-theme, it should only affect html

Reply:
cool, have added check for `standaloneHTMLBuilder`.